api/tests/integration/infrastructure/repositories/pole-emploi-sending-repository_test.js

```diff
@@ -74,28 +75,44 @@ describe('Integration | Repository | PoleEmploiSending', () => {
       settings.poleEmploi.poleEmploiSendingsLimit = originalEnvPoleEmploiSendingsLimit;
     });
 
-    it('should render 3 sendings because of the poleEmploiSendingsLimit variable', async () => {
-      // when
-      const sendings = await poleEmploiSendingRepository.get();
+    context('when there is no cursor', function() {
+      it('should render 3 sendings because of the poleEmploiSendingsLimit variable', async () => {
```

Comment:
**suggestion (non-blocking) :** L'enchainement du `when there is no cursor` avec ce it laisse penser que la limite n'est utilisée que quand il n'y a pas de curseur